src/Microsoft.Health.Fhir.Core/Data/GraphQL/patient.graphql

```diff
@@ -0,0 +1,75 @@
+﻿# FHIR GraphQL Schema. Version 4.6.0
+
+# import the types from 'types.graphql'
```

Comment:
Could you add a comment on why some of the Patient properties are still commented out? I think it would be helpful for anyone looking at this latter to understand why those properties don't work.

Reply:
Good point RB, I added some comments in the "Patient.graphql"file